### DISCUSSION TITLE
frame drop after last update custom FBW

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4020] [General]
Hi simmers,

after the last update from the custom FBW this morning, I regognize huge frame drops! Looks like ATC produces the drops and some kind of clouds (CB). But I am wondering cause I never had this problems with ATC and clouds before. The frames are dropping from 45 to 15 ... :-( ...

Anybody else have problems like this?

Greets Klaus

**Comment by robinottawa** [comment]
Are you sure it's just this FBW update? There's a ton of complaints about the last MSFS update causing the same thing. It's recognised by zendesk.

**Reply by CptBayer** [reply]
Hi,

it seems to be, cause after the last MS update, I flew a lot of legs without problems.... and right after last custom FBW update I got frame drops that I never had before. But you are right, there is a possibility that this could have other reason coming from the MS himself. It is a big world of code.... so ... 

Before the last custom FBW update I flew inbound FOOL with a lot of clouds and no problems. It was a 4 hour leg. The next day after the update taking off from FOOL without clouds but with massiv frame drops.. It was a bit mysterious ...

Greets Klaus

**Reply by robinottawa** [reply]
I'm trying the Dev version today. 

EDIT: after a 90 minute short haul in the dev, I had now normal fps drops (from ~25 to ~15-20) and normal glitches in AP.

**Reply by CptBayer** [reply]
Hi Robin,

you have only 25 fps ?! My drops are from round about 45 to 15 as mentioned above ..... that is nearl 66 % ...

Greets klaus

**Reply by robinottawa** [reply]
Yeah, it makes us even. ;)   

But I wish I had better rates before too, so I could look forward to getting them back after they fix this. I can still fly but it takes a lot of the fun out of it knowing that my mid-level gaming computer is useless unless I turn off all the scenery. :(

Cheers
